deploy/11_layer_zero.ts

```diff
@@ -32,11 +32,17 @@ const func: DeployFunction = async function (hre1: HardhatRuntimeEnvironment) {
   if (currentNetworkType == NetworkType.local && lzEndpoint == zeroAddress) {
     lzEndpoint = (await hre.getNamedAccounts()).lzEndpoint;
     const mockLZEndpoint = await hre.deployments.deploy('MockLZEndpoint', {
-      from: lzEndpoint,
+      ...(await txParams({
+        hre,
+        from: lzEndpoint,
+        to: '0x0000000000000000000000000000000000000000',
```

Comment:
is this is just caching current gas parameters?

Reply:
This function puts together all the data needed. Nonce, gasPrice, and gasLimit.